src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  onClose: (todo: Todo | '') => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [isLoad, setIsLoad] = useState(true);
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(todo.userId).then(userFromServer => {
+      setUser(userFromServer);
+      setIsLoad(false);
+    });
+  }, []);
```

Comment:
Fix it 
<img width="960" alt="image" src="https://github.com/user-attachments/assets/e3b0f3d6-de43-4765-9739-fd4de0a359ba" />